e2e/tests/overview-page.spec.ts

```diff
@@ -241,6 +241,28 @@ test('Export to CSV button fails to request all features and shows toast', async
   await toast.waitFor({state: 'visible'});
 });
 
+test('Test id search atoms in a query', async ({page}) => {
+  await gotoOverviewPageUrl(page, 'http://localhost:5555/');
+  const searchbox = page.locator('#inputfield');
+  await expect(searchbox).toBeVisible();
+  await expect(searchbox).toHaveAttribute('value', '');
+
+  const stats = await page.locator('.stats-summary');
```

Comment:
I know we have some existing usage of using page.locator but we should use the [recommended locators](https://playwright.dev/docs/locators#:~:text=These%20are%20the%20recommended%20built%2Din%20locators) when possible.
```suggestion
  const regex = /(\d+) features/;
  const stats = page.getByText(regex);
```